addons/portal/controllers/portal.py

```diff
@@ -560,9 +615,9 @@ def _show_report(self, model, report_type, report_ref, download=False):
             reporthttpheaders.append(('Content-Disposition', content_disposition(filename)))
         return request.make_response(report, headers=reporthttpheaders)
 
-def get_error(e, path=''):
+def get_message(e, path=''):
```

Comment:
Why renaming ? I think this whole error management stuff deserves a cleaning, but outside the scope of this task ... would keep it as it is ^^